evm/governance.js

```diff
@@ -0,0 +1,285 @@
+'use strict';
+
+require('dotenv').config();
+
+const {
+    Wallet,
+    getDefaultProvider,
+    utils: { isAddress, defaultAbiCoder, keccak256, Interface },
+    Contract,
+    BigNumber,
+} = require('ethers');
+const readlineSync = require('readline-sync');
+const { Command, Option } = require('commander');
+
+const {
+    printInfo,
+    printWalletInfo,
+    loadConfig,
+    isNumber,
+    isValidTimeFormat,
+    etaToUnixTimestamp,
+    getCurrentTimeInSeconds,
+    wasEventEmitted,
+    printWarn,
+    printError,
+} = require('./utils');
+const IGovernance = require('@axelar-network/axelar-gmp-sdk-solidity/interfaces/IAxelarServiceGovernance.json');
+
+async function processCommand(options, chain) {
+    const { contractName, address, governanceAction, calldata, nativeValue, eta, privateKey, yes } = options;
+
+    if (contractName !== 'AxelarServiceGovernance' && contractName !== 'InterchainGovernance') {
```

Comment:
If these are the only 2 names then we can add them as choices in the `contractName` option so won't have to check for any other value.

Reply:
yeah I like that idea